tests/integration/components/cf-field-test.js

```diff
@@ -105,20 +105,6 @@ module("Integration | Component | cf-field", function (hooks) {
     assert.dom("input[type=text]").hasClass("uk-disabled");
   });
 
-  test("it validates input", async function (assert) {
-    assert.expect(1);
-
-    await render(hbs`{{cf-field field=field}}`);
-
-    await fillIn("input", "Test");
-
-    assert
-      .dom("span.validation-errors")
-      .hasText(
-        't:caluma.form.validation.tooLong:("max":2,"min":1,"value":"Test")'
-      );
-  });
```

Comment:
Why?

Reply:
Since maxlength is now used as attribute on the input we couldn't fill in that value. I now updated the validation so the test makes more sense.